index.html

```diff
@@ -127,7 +127,6 @@
             'error', loadErrHandler, true /* capture phase type of listener */);
     </script>
     <script><!--#include virtual="/config.js" --></script><!-- adapt to your needs, i.e. set hosts and bosh path -->
-    <script src="static/utils.js?v=1"></script>
```

Comment:
That's part of a subject that @hristoterezov brought up to you in a meeting between the two of you. He alerted me as well to it yesterday. Removing static/utils.js (potentially) bundles its equivalent twice in the app: once in do_external_connect.min.js and a second time in app.bundle.min.js. I believe you as the PR author should have alerted the rest of us to the consequences if not during the design phase before actually writing the source code, then at least after the fact.